### PR TITLE
1125 implement bootstrap dropdown

### PR DESCRIPTION
**Pull request by ElenaPoduzova**
[#1225](https://github.com/jdi-testing/jdi-light/issues/1125)

### CODE REVIEW

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/dropdown/DropdownFormTest.java [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.dropdownForm;
+import static io.github.epam.bootstrap.tests.BaseValidations.baseValidation;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+import static org.hamcrest.Matchers.is;
+
+public class DropdownFormTest extends TestsInit {
+
+    private static FormContacts USER = new FormContacts().set(c -> {
+        c.email = "testEmail@test.com";
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/dropdown/DropdownFormTest.java]: possibly it's better to use example.com

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/dropdown/DropdownFormTest.java [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    @Test(dataProvider = "linksDataProvider")
+    public void linkItemsTest(String itemText, String itemHref) {
+        dropdownForm.expand();
+
+        int currWindowNum = WindowsManager.windowsCount();
+        dropdownForm.select(itemText);
+
+        WindowsManager.switchToWindow(currWindowNum + 1);
+        String url = WebPage.getUrl();
+        Assert.assertEquals(url, itemHref);
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/dropdown/DropdownFormTest.java]: assertEquals(url, itemHref) (without Assert , in other places we use this style)

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/dropdown/DropdownFormTest.java [hunk 3]
```diff
@@ -0,0 +1,170 @@
+package io.github.epam.bootstrap.tests.complex.dropdown;
+
+import com.epam.jdi.light.elements.common.WindowsManager;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.ui.bootstrap.elements.common.TextField;
+import io.github.com.entities.FormContacts;
+import io.github.epam.TestsInit;
+import org.testng.Assert;
```
Comment (elv1s42) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/dropdown/DropdownFormTest.java]: Codacy Issue found: [Avoid unused imports such as 'org.testng.Assert'](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4481601)

**Comment by elv1s42** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/dropdown/DropdownTests.java  1
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/dropdown/DropdownOptionsTests.java  2
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/dropdown/DropdownFormTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4481601&bid=15122294)

**Comment by elv1s42** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/dropdown/DropdownFormTest.java  1
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/dropdown/DropdownTests.java  1
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/complex/dropdown/DropdownOptionsTests.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4481601&bid=15122294)

**Pull request closed without merging**